src/Cohesity.Powershell/Scripts/DownloadFile/Download-CohesityFilesAndFolders.ps1

```diff
@@ -0,0 +1,106 @@
+function Download-CohesityFilesAndFolders {
+    <#
+        .SYNOPSIS
+        Request to download the specified file/folder from the specified server.
+        .DESCRIPTION
+        The Download-CohesityFilesAndFolders function is used to download specific file/folder using REST API from the specified server under tthe specified target folder.
+        .NOTES
+        If target path is not specified, then file/folder will be downloaded under home folder
+        .EXAMPLE
+        Download-CohesityFilesAndFolders -FileName <FileFolderName> -ServerName <ServerName> -OutFile <TargetPath>
```

Comment:
Give concrete example of
1. file 
2. folder 
Write the assumptions we spoke about : 
a. latest snapshot 
b. the first match if there are multiple matches.